provisioning/templates-api/kubernetes/templates/etcd/values.yaml

```diff
@@ -98,7 +98,7 @@ defragmentation:
     ## @param defragmentation.cronjob.schedule Schedule in Cron format to run defragmentation
     ## See https://en.wikipedia.org/wiki/Cron
     ##
-    schedule: "* */24 * * *"
+    schedule: "0 */24 * * *"
```

Comment:
Nahodou som si vsimol, ze je tam tych metrik nejak vela.
A ono to bezalo kazdu 24h, kazdu minutu, tj. od `00:00-01:00` - `60x`.

Before:
![image](https://user-images.githubusercontent.com/19371734/200331188-8fb9b831-2d8a-4bbb-bbc2-ff43390efb6a.png)


After:
![image](https://user-images.githubusercontent.com/19371734/200331060-c4cc959d-2d4d-4afd-b5fa-d2e1ad81825a.png)